includes/handler/class-undo.php

```diff
@@ -40,7 +40,9 @@ public static function handle_undo( $activity ) {
 			return;
 		}
 
-		$type = $activity['object']['type'];
+		$type    = $activity['object']['type'];
+		$state   = false;
+		$user_id = null;
```

Comment:
Should this be 0? It's documented as `@param int   $user_id` below

Reply:
I think I have to update the phpdoc then, because we use `0` for the blog-user (I'm no longer sure whether that was a good idea).

Reply:
0 means [there's no logged-in user](https://developer.wordpress.org/reference/functions/get_current_user_id/#return). So I guess it depends what's being done with it.